src/Invoice/Calculator/WeeklyInvoiceCalculator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Kimai time-tracking app.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Invoice\Calculator;
+
+use App\Invoice\CalculatorInterface;
+use App\Invoice\InvoiceItemInterface;
+
+/**
+ * A calculator that sums up the invoice item records per week.
+ */
+class WeeklyInvoiceCalculator extends AbstractSumInvoiceCalculator implements CalculatorInterface
+{
+    protected function calculateSumIdentifier(InvoiceItemInterface $invoiceItem): string
+    {
+        if (null === $invoiceItem->getUser()->getId()) {
```

Comment:
I think this `if` can be removed

Reply:
fixed in b4394da003fd1e9f9ab4654ee6461a9c5b511ffe